services/cd-service/pkg/repository/transformer.go

```diff
@@ -58,18 +63,58 @@ func releasesDirectoryWithVersion(fs billy.Filesystem, application string, versi
 	return fs.Join(releasesDirectory(fs, application), versionToString(version))
 }
 
+func GetEnvironmentLocksCount(fs billy.Filesystem, env string) int {
+	envLocksCount := 0
+	envDir := environmentDirectory(fs, env)
+	locksDir := fs.Join(envDir, "locks")
+	if entries, _ := fs.ReadDir(locksDir); entries != nil {
+		envLocksCount += len(entries)
+	}
+	return envLocksCount
+}
+
+func GetEnvironmentApplicationLocksCount(fs billy.Filesystem, environment, application string) int {
+	envAppLocksCount := 0
+	appDir := environmentApplicationDirectory(fs, environment, application)
+	locksDir := fs.Join(appDir, "locks")
+	if entries, _ := fs.ReadDir(locksDir); entries != nil {
+		envAppLocksCount += len(entries)
+	}
+	return envAppLocksCount
+}
+
+func UpdateDatadogMetrics(fs billy.Filesystem) error {
```

Comment:
even if it's trivial, I would like to have one tests that checks that this function does not crash when ddMetrics is nil.

Reply:
I believe it will return immediately with nil if ddMetrics is nil

Reply:
Yes. I want to ensure that nobody removes that line by accident.

Reply:
(but I'm also ok if you do that later)